README.md

```diff
@@ -1,76 +1,231 @@
+    <a href="https://github.com/Zagrios/bs-manager/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/Zagrios/bs-manager?style=for-the-badge"></a>
+    <br>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-DISCORD-5865f2?style=for-the-badge&logo=discord&logoColor=ffffff" alt="discord"/></a>
+    <a href="https://mee6.xyz/fr/m/bsmanager"><img src ="https://img.shields.io/badge/-🥰%20Support%20BSM-EC4546?style=for-the-badge" alt="Donation"/></a>
+    <a href="https://discord.gg/uSqbHVpKdV"><img src ="https://img.shields.io/badge/-Twitter-F5F8FA?style=for-the-badge&logo=Twitter" alt="Twitter"/></a>
+    <br>
+    [<a href="#readme-bot">go to bottom</a>]
+</div>
+
+---
+<!-- TABLE OF CONTENTS -->
+<b>Table of Contents</b>
 <ul>
-	<li><a href="https://github.com/Zagrios">Zagrios</a> - Founder, Lead Developer & Project Lead</li>
-	<li><a href="https://github.com/Iluhadesu">Iluhadesu</a> - Custom DepotDownloader Developer</li>
-	<li><a href="https://github.com/GaetanGrd">Gaëtan Gerard</a> - Main Alpha Tester</li>
+    <li>
+        <a href="#about-the-project">About The Project</a>
+        <ul>
+            <li><a href="#built-with">Built With</a></li>
+            <li><a href="#contributing">Contributing</a></li>
+        </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <ul>
+        <li><a href="#how-to-install">How to install?</a></li>
+        <li><a href="#manage-versions">Manage Versions</a></li>
+        <li><a href="#customize-your-instances">Customize your versions</a></li>
+        <li><a href="#install-mods">Install Mods</a></li>
+        <li><a href="#install-maps">Install Maps</a></li>
+        <li><a href="#one-click">One Click</a></li>
+        <li><a href="#having-problems">Having problems?</a></li>
+    </ul>
 </ul>
-<p align="center">
-	<img src="https://github.com/Zagrios/bs-manager/blob/master/resources/readme/beat-conflict.png?raw=true" width="130"/>
-</p>
 
+<!-- ABOUT THE PROJECT -->
+<div align="center">
+    <h1> </h1>
+    <h1><b>About The Project</b></h1>
+</div>
+<div>
+    <a href="https://github.com/Zagrios/bs-manager">BSManager</a> was created to unify and improve the two most used tools in the community (<a href="https://github.com/RiskiVR/BSLegacyLauncher">BS legacy launcher</a> and <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>).
+    <br><br>
+    <h4><a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to:</h4>
+    <ul>
+        <li><b>Manage instances of multiple versions</b>, designed for people who need to play on different versions or have multiple profiles for the same version.</li>
+        <li><b>Download your maps in the same way</b>, in addition to that, the maps are not version dependent, so you will be able to find your maps whatever version you are using without having to transfer them.</li>
```

Comment:
🔴  depending on your choice maps are not version dependent

Reply:
```suggestion
        <li><b>Download your maps in the same way</b>, in addition to that, depending on your choice, maps are not version dependent. so you will be able to find your maps whatever version you are using without having to transfer them.</li>
```